scripts/visualize.py

```diff
@@ -43,59 +42,35 @@ def main(_):
     if random_seed is not None:
         torch.manual_seed(random_seed)
 
-    dataset = meshnets.utils.datasets.FromDiskGeometricDataset(FLAGS.data_dir)
-    size_dataset = len(dataset)
-    num_training = int(FLAGS.train_split * size_dataset)
-    _, validation_dataset = torch.utils.data.random_split(
-        dataset, [num_training, size_dataset - num_training])
-
-    wrapper = meshnets.utils.model_loading.load_model_from_mlflow(
-        FLAGS.tracking_uri, FLAGS.run_id, FLAGS.checkpoint)
-
-    sample_idx = np.random.choice(validation_dataset.indices)
-
-    mesh_path = dataset.get_mesh_path(sample_idx)
-    graph_path = dataset.get_graph_path(sample_idx)
+    dataset = datasets.load_dataset(
+        "inductiva/wind_tunnel",
+        version=FLAGS.dataset_version,
+        split=f"train[-{1 - FLAGS.train_split:.0%}:]",
+        download_mode="force_redownload")
+    len_dataset = len(dataset)
+    random_example = dataset[np.random.randint(len_dataset)]
+    random_example = data_mappers.to_undirected(random_example)
+    random_example = data_mappers.make_edge_features(random_example)
+    random_example = data_mappers.make_node_features(random_example)
+
+    wrapper = model_loading.load_model_from_mlflow(FLAGS.tracking_uri,
```

Comment:
This wrapper refers to the model itself no?

Reply:
Kinda. It is called wrapper because this actually loads the whole `pytorch_lightning` [class](https://github.com/inductiva/meshnets/blob/63fe3db029136f4af6c19dde5a91827e9bf8b576/meshnets/modules/lightning_wrapper.py#L11) that also contains most training.

The itself [receives as argument](https://github.com/inductiva/meshnets/blob/63fe3db029136f4af6c19dde5a91827e9bf8b576/meshnets/modules/lightning_wrapper.py#L15) a pytorch model